app/views/layouts/main_layout.html.erb

```diff
@@ -1,14 +1,16 @@
 <% content_for :main do %>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-one-third">
-    <ul class="govuk-list">
-      <li>
-        <a class="govuk-link" href="#">Manage certificates</a>
-      </li>
-      <li>
-        <a class="govuk-link" href="#">Team Members</a>
-      </li>
-    </ul>
+    <div class="side-navigation">
+      <ul class="govuk-list">
+        <li class="active">
```

Comment:
I wouldn't hard code these in the main layout, they should be dynamic, same with the `--no-visited-state` below
```suggestion
        <li>
```

Reply:
Do you mean something like this `<li class="<%= 'active' if current_page?(dashboard) %>">`?

Reply:
yep!

Reply:
With the `--non-visited-state`, we actually want that hard coded in there for the styling to match the prototype. Once the page is visited the link changes colour, but in the prototype they seem to want to keep it the same colour and just bold the current page. This is something i will run by Chris

Reply:
Ah yeah, you're right. The guidance says to use the class when `where it is not helpful to distinguish between visited and unvisited states, for example when linking to pages with frequently-changing content such as the dashboard for an admin interface.`